extensions/iceberg/build.gradle

```diff
@@ -30,7 +30,6 @@ dependencies {
     implementation libs.iceberg.core
     implementation libs.iceberg.bundled.guava
 
-    implementation libs.hadoop.hdfs.client
     // Also inherits hadoop-common, see io.deephaven.hadoop-common-dependencies
```

Comment:
Does this comment still hold? That is, should we delete it, too?

Reply:
Yes, this comment still holds because we have the `id 'io.deephaven.hadoop-common-dependencies'` at the top which pulls in hadoop-common.